doc/sailfish-webview-webview.qdoc

```diff
@@ -14,4 +14,20 @@
   import namespace.
 */
 
+/*!
+  \qmlproperty PopupProvider WebView::popupProvider
+  \brief A provider of custom user interaction components
+
+  Client applications can define their own custom user interaction components
+  which extend the various interface types defined in the
+  \l{Sailfish.WebView.Popups} namespace.
+
+  In order for those custom components to be used by the WebView, the client
+  must register a \c{PopupProvider} instance (which specifies the various
+  custom components which should be used) with the WebView via this property.
+
+  It is an optional property.
+
+  \sa {Popup Customization}, custompopups
```

Comment:
If property link doesn't work, this see also could be like...

```
\sa {Popup Customization}, PopupProvider, custompopups
```

Reply:
done